Library/.rubocop.yml

```diff
@@ -8,7 +8,17 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+Homebrew/FormulaCop:
+  Enabled: false
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
If this will need to be repeated for every cop I think the existing approach was superior.